db/queries/scores/__init__.py

```diff
@@ -0,0 +1,8 @@
+# flake8: noqa
+from .queries import get_latest_just_score_for_application_sub_crit
+from .queries import create_just_score_for_application_sub_crit
+
+__all__ = [
+    get_latest_just_score_for_application_sub_crit,
+    create_just_score_for_application_sub_crit,
```

Comment:
⛏️ `__all__` takes a list of strings.
```suggestion
    "get_latest_just_score_for_application_sub_crit",
    "create_just_score_for_application_sub_crit",
```